index.js

```diff
@@ -7,23 +7,31 @@ const cleanCSS = require('gulp-clean-css');
 const prefix = require('gulp-autoprefixer');
 const chalk = require('chalk');
 const _debounce = require('lodash.debounce');
-
+const path = require('path');
+const map = require('map-stream');
+const remap = function(file, cb) {
+    // console.log(file.path);
+    file.path = path.parse(file.path).base;
+    cb(null, file);
+};
 const PLUGIN_NAME = 'Eleventy-Plugin-SASS';
-
+const PLUGIN_SHORT = 'PS';
```

Comment:
```suggestion
const PLUGIN_SHORT = 'EPS';
```

Just an idea but PS was confusing as I read it as a variable for Plugin Short. This is more explicit to the actual plugin name